src/idlcxx/tests/streamer_generator.c

```diff
@@ -1356,10 +1356,8 @@ void generate_array_instance_funcs(idl_ostream_t* ostr, bool ns)
 "  }\n\n"\
 "  size_t typedef_key_max_size_td_6(const td_6 &obj, size_t position)\n"\
 "  {\n"\
-"    if (position != UINT_MAX)   position += (4 - (position&0x3))&0x3;  //alignment\n"\
-"    if (position != UINT_MAX)   position += 4;  //bytes for sequence entries\n"\
-"    position = UINT_MAX;\n"\
-"    return position;\n"\
+"    (void)position;\n"\
+"    return UINT_MAX;\n"\
```

Comment:
Why is position explicitly ignored, while obj, which is also not touched is not explicitly ignored?
I realize that this is a testcase, but I guess that the string in question reflects the code that is actually generated?

Reply:
You are correct, this is caused by a shortcoming in my code that does not see the difference between the max size function of an object instance and a typedef definition.
I will look into a fix for this